gulpfile.js

```diff
@@ -1,7 +1,89 @@
-var gulp = require('gulp');
+var gulp          = require('gulp');
+var runSequence   = require('run-sequence');
+var jspm          = require('gulp-jspm');
+var jade          = require('gulp-jade');
+var s3            = require('gulp-s3');
+var rename        = require('gulp-rename');
+var fs            = require('fs');
 
-gulp.task('default', ['builder']);
+gulp.task('default', ['buildDevIndex']);
 
-gulp.task('builder', function(cb) {
-  
+gulp.task('buildDevIndex', function (callback) {
+  var YOUR_LOCALS = { release: false };
+
+  return gulp
+    .src('./index.jade')
+    .pipe(jade({ locals: YOUR_LOCALS }))
+    .pipe(gulp.dest('./'));
+});
+
+gulp.task('buildIndex', function (callback) {
+  var YOUR_LOCALS = { release: true };
+
+  return gulp
+    .src('./index.jade')
+    .pipe(jade({ locals: YOUR_LOCALS }))
+    .pipe(gulp.dest('./.dist'));
+});
+
+gulp.task('jspmBundleSfx', function (callback) {
+  return gulp
+    .src('./app/main.ts')
+    .pipe(jspm({ selfExecutingBundle: true, minify: true, mangle: false }))
```

Comment:
Nice to have: use `gulp-sourcemaps` around this.  The `gulp-jspm` docs say they support it.